modules/veteran_verification/app/models/veteran_verification/disability_rating.rb

```diff
@@ -15,7 +15,11 @@ class DisabilityRating
     attribute :individual_ratings, Array
 
     def self.rating_service
-      BGS::DisabilityRatingService.new
+      if Settings.bgs.mock_bgs == true
```

Comment:
Should this be a setting under vet_verification or renamed to reflect that it's only mocking the disability rating service?